projects/ardin_mb/ardin_mb.cc

```diff
@@ -32,109 +32,27 @@
 
 using namespace BoBRobotics;
 
-//----------------------------------------------------------------------------
-// Anonymous namespace
-//----------------------------------------------------------------------------
-namespace
-{
-void keyCallback(GLFWwindow *window, int key, int, int action, int)
-{
-    // If action isn't a press or a release, do nothing
-    if(action != GLFW_PRESS && action != GLFW_RELEASE) {
-        return;
-    }
-
-    // Determine what state key bit should be set to
-    const bool newKeyState = (action == GLFW_PRESS);
-
-    // Extract key bitset from window's user pointer
-    StateHandler *stateHandler = reinterpret_cast<StateHandler*>(glfwGetWindowUserPointer(window));
-
-    // Apply new key state to bits of key bits
-    switch(key) {
-        case GLFW_KEY_LEFT:
-            stateHandler->setKeyState(StateHandler::KeyLeft, newKeyState);
-            break;
-
-        case GLFW_KEY_RIGHT:
-            stateHandler->setKeyState(StateHandler::KeyRight, newKeyState);
-            break;
-
-        case GLFW_KEY_UP:
-            stateHandler->setKeyState(StateHandler::KeyUp, newKeyState);
-            break;
-
-        case GLFW_KEY_DOWN:
-            stateHandler->setKeyState(StateHandler::KeyDown, newKeyState);
-            break;
-
-        case GLFW_KEY_R:
-            stateHandler->setKeyState(StateHandler::KeyReset, newKeyState);
-            break;
-
-        case GLFW_KEY_SPACE:
-            stateHandler->setKeyState(StateHandler::KeyTrainSnapshot, newKeyState);
-            break;
-
-        case GLFW_KEY_ENTER:
-            stateHandler->setKeyState(StateHandler::KeyTestSnapshot, newKeyState);
-            break;
-
-        case GLFW_KEY_S:
-            stateHandler->setKeyState(StateHandler::KeySaveSnapshot, newKeyState);
-            break;
-
-        case GLFW_KEY_W:
-            stateHandler->setKeyState(StateHandler::KeyRandomWalk, newKeyState);
-            break;
-
-        case GLFW_KEY_V:
-            stateHandler->setKeyState(StateHandler::KeyBuildVectorField, newKeyState);
-            break;
-    }
-}
-//----------------------------------------------------------------------------
-void handleGLFWError(int errorNumber, const char *message)
-{
-    LOGE << "GLFW error number: " << errorNumber << ", message:" << message;
-}
-}   // anonymous namespace
-
-
-
 int main(int argc, char *argv[])
 {
-    // Set GLFW error callback
-    glfwSetErrorCallback(handleGLFWError);
+    sf::ContextSettings settings;
+    settings.antialiasingLevel = 8;
```

Comment:
Is there any particular reason you're doing this?

Reply:
I just copied and pasted it from somewhere where I was using anti-aliasing. Would you rather it was turned off?

Reply:
As these examples are using OpenGL directly rather than SFML, I just suspect it doesn't do anything :)

Reply:
Ah fair enough. I'll drop it then.